README.md

```diff
@@ -1 +1,226 @@
+	* if not specified: autogenerated
+* hawkular-cassandra-ca.cert
+	* The certificate for the CA used to sign the hawkular-cassandra.pem
+	* required if hawkular-cassandra-ca.cert is specified, ignored otherwise
+* heapster.cert
+	* The certificate used by Heapster
+	* if not specified: autogenerated
+* heapster.key
+	* The key to be used with the heapster certificate
+	* only required if heapster.cert is specified, ignored otherwise
+* heapster_client_ca.cert
+	* The certificate authority certificate used to generate heapster.cert
+	* required if heapster.cert is specified, otherwise set to an autogenerated ca certificate
+* heapster_allowed_users
+	* A file containing a comma separated list of CN to accept from certificates signed with the specified CA
+	* required if heapster.cert is specified, otherwise set to no allowed users
+	
+If the secrets to be used are placed all within a single directory, the following command will create the secret for you:
+
+	oc secrets new metrics-deployer path/to/dir
+	
+If the secrets are not all located within a single directory, the following command can be used to specify the location of the files:
+
+	oc secrets new metric-deployer hawkular-metrics.pem=/my/dir/hm.pem \
+	                               hawkular-metrics-ca.cert=/my/dir/hm-ca.cert
+
+
+### Deploying the Metrics Components
+
+To deploy the metric components, you will need to deploy the 'metrics' template.
+
+You will need to use the same IMAGE_PREFIX and IMAGE_VERSION used to build the containers:
+
+	oc process -f metrics.yaml -v \
+	HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,IMAGE_PREFIX=openshift/origin-,IMAGE_VERSION=devel \
+	| oc create -f -
+	
+#### Template Parameters
+
+The following are the various parameters the template will accept
+
+#####IMAGE_PREFX
+The prefix selected when the containers were built.
+
+#####IMAGE_VERSION
+The version selected when the containers were built.
+
+##### HAWKULAR_METRICS_HOSTNAME
+The hostname that hawkular metrics is going to be hosted under. This is used to generate the Hawkular Metrics certificate and is used for the host in the route configuration.
+
+##### REDEPLOY
+If the redeploy parameter is set to `true` it will delete all the components, service accounts, secrets and persistent volume claim. This will permanently delete any metrics which are stored.
+
+##### MASTER_URL
+The url to used for components to access the kubernetes master. Defaults to `https://kubernetes.default.svc.cluster.local:443`
+
+##### CASSANDRA_NODES
+How many initial Cassandra nodes should be deployed. This defaults to a single node cluster.
+
+##### CASSANDRA_PV_SIZE
+The requested size that each Cassandra node is requesting. This defaults to 1gi
+
+##### METRIC_DURATION
+How many days that metrics should be stored for. This defaults to 7 days.
+
+	
+### Cassandra Scaling
+
+Since the Cassandra nodes use persistent storage, we cannot currently scale up or down using replication controllers.
```

Comment:
This is temporary right?  we should probably mention that and that it will be addressed in subsequent releases.